docs/apis-tools/tasklist-api-rest/controllers/tasklist-api-rest-form-controller.md

```diff
@@ -14,35 +14,40 @@ Get the form details by `formId` and `processDefinitionKey` required parameters.
 
 #### URL
 
-`/v1/forms/{formId}?processDefinitionKey={processDefinitionKey}`
+`/v1/forms/{formId}?processDefinitionKey={processDefinitionKey}&version={formVersion}`
 
 #### Method
 
 `GET`
 
 #### Request parameters
 
-| Parameter name       | Type  | Required | Description                         |
-| -------------------- | ----- | -------- | ----------------------------------- |
-| formId               | path  | `true`   | ID of the form                      |
-| processDefinitionKey | query | `true`   | Reference to the process definition |
+| Parameter name       | Type  | Required | Description                                        |
+| -------------------- | ----- | -------- | -------------------------------------------------- |
+| formId               | path  | `true`   | ID of the form                                     |
+| processDefinitionKey | query | `true`   | Reference to the process definition                |
+| version              | query | `false`  | Version of the form, valid only for deployed forms |
 
 :::caution
 The `formId` is a value generated by the internal Tasklist API and is distinct from the ID specified in the form editor. While the key returned to the API user when getting a task follows the format `camunda-forms:bpmn:<formId>`, only the `<formId>` segment is required to retrieve the form schema.
```

Comment:
> a task follows the format `camunda-forms:bpmn:<formId>`

Is that also the case for deployed forms? Should we update this section to discern embedded and deployed?

Reply:
I updated this on this commit: [
](https://github.com/camunda/camunda-docs/pull/2851/commits/fa47609ca55c0363c9b1d6a7e73a4ad9605bd9d4) 

Also I changed the box to "info" as it is more appropriated. If you think "Caution" is better, I can change again 👍